wiki/SimRel/2025-03.md

```diff
@@ -13,12 +13,12 @@ This documents related to 2025-03, the Eclipse Foundation's multi-project [Simul
 | **2025-03 M3** | Friday, February 21, 2025 | 02/14 to 02/21 | <ul><li>IP Log submission deadline</ul> | 3 weeks from M2 |
 | **2025-03 RC1** | Friday, February 28, 2025 | 02/21 to 02/28 | <ul><li><b>No new features and APIs after this date!</b><li>Release Review materials due<li>New and Noteworthy entries due</ul> | 1 week from M3 |
 | **2025-03 RC2** | Friday, March 7, 2025 | 02/28 to 03/07 | | 1 week from RC1 |
-| **Quiet period** | | 03/07 to 03/11 | | No builds. It is assumed all code is done by end of RC2. |
+| **Quiet period** | | 03/08 to 03/11 | | No builds. It is assumed all code is done by end of RC2. |
 | **2025-03 GA** | Wednesday, March 12, 2025 | | <ul><li>Release reviews conclude on this date</ul> | 5 days from RC2 |
 
-[Google Calendar Link](https://calendar.google.com/calendar/embed?src=gchs7nm4nvpm837469ddj9tjlk@group.calendar.google.com&dates=20241201%2F20250331&hl=en&mode=AGENDA)
+[Google Calendar Link](https://calendar.google.com/calendar/embed?src=gchs7nm4nvpm837469ddj9tjlk@group.calendar.google.com&dates=20241227%2F20250313&hl=en&mode=AGENDA)
 
 ## Non-wiki pages related to the release
 
-[List of participating projects](https://projects.eclipse.org/releases/2025-03)
+[List of participating projects](https://www.eclipse.org/projects/releases/2025-03)
```

Comment:
This seems like a broken link.

This thing is a redirection:

https://www.eclipse.org/projects/releases/

Reply:
I've updated the markdown files and template to use `https://projects.eclipse.org/releases/${SIMREL_NAME}`.
It seems these links are broken in the overview for `2024-09` and `2024-12` at the moment. I cannot tell how they made it into the repo.

The links in the template can never work. So we should exclude it from the check somehow, but I didn't found an easy option the action used for.
Btw. https://github.com/gaurav-nelson/github-action-markdown-link-check also says it's deprecated, so we should probably migrate to the recommended replacement.

Reply:
Ignoring is fine in my opinion.